src/lib/Translation/Extractor/LimitationTranslationExtractor.php

```diff
@@ -33,8 +35,13 @@ public function __construct(array $policyMap)
         $this->policyMap = $policyMap;
     }
 
+    /**
+     * @deprecated Since ibexa/admin-ui 4.4: The method "LimitationTranslationExtractor::extract()" method is deprecated, will be removed in 5.0.
+     */
     public function extract()
     {
+        @trigger_deprecation('ibexa/admin', '4.4', 'The %s() method is deprecated, will be removed in 5.0.', __METHOD__);
```

Comment:
`@` are not necessary to silence this function.
```suggestion
        trigger_deprecation('ibexa/admin', '4.4', 'The %s() method is deprecated, will be removed in 5.0.', __METHOD__);
```